copied apy calculation code from solend-lite to sdk (calculation was broken)

silostack opened the pull request:
the calculated supply apy was way off from what's shown on lite.solend.fi and solend.fi. copied the code over from the lite.solend.fi code to the sdk.

nope-finance commented:
hm so i was looking at the code and i think issue was also that the rates calcs are duplicated in 
`solend-sdk/src/core/utils/rates.ts` i think it would be preferable to just remove these functions or have them reference the functions in rates.ts instead of copy pasting the logic

silostack commented:
> hm so i was looking at the code and i think issue was also that the rates calcs are duplicated in `solend-sdk/src/core/utils/rates.ts` i think it would be preferable to just remove these functions or have them reference the functions in rates.ts instead of copy pasting the logic

Yea I was in a hurry to get things fixed so copy/pasted lol. I updated the code and fixed in a couple other places in the same file as well. There's still a lot more code that's duped and can get cleaned up in that file...

nope-finance commented on solend-sdk/src/classes/reserve.ts:
```diff
@@ -29,70 +29,83 @@ export class SolendReserve {
   }
 
-  private calculateBorrowAPR(reserve: ParsedReserve) {
+  private calculateBorrowAPR = (reserve: ParsedReserve) => {
```
i think this as caclulateUtilizationRatio are also in that other file

nope-finance replied:
i'd just edit it but i think i can't edit a fork

silostack replied:
Removed that method along with the caclulateUtilizationRatio from the reserve.ts file as they're no longer used.

nope-finance replied:
i think possible one might want to grab the utilization ratio but yeh seems fine now ill merge this

nope-finance replied:
actually kinda same with calculateBorrowAPR one probably would want it
